### DISCUSSION TITLE
Question about 1.20

### DISCUSSION THREAD

**Discussion by vico93** [cc-tweaked/CC-Tweaked#1441] [Q&A] [answered]
Will a build for 1.20 (on Fabric of course) be developed only when Mojang officially releases it or one will be released for pre1?

Since it seems to be a huge update why not release at least one update for the pre1 version, even to get acquainted with the internal changes of the new version?

Sorry if the question sounds annoying, not my intention.

**Comment by SquidDev** [comment, marked as answer]
I'm not in any particular rush to update to 1.20, so won't be releasing snapshot non-release builds of any form.

**Reply by vico93** [reply]
No problem.

I also trying to build it from IDEA. How can i build the fabric version?

**Reply by SquidDev** [reply]
`./gradlew :fabric:assemble` is probably the easiest way. If you want to test in IDEA, Loom _should_ create run configurations automatically, though you may need to restart IDEA for them to show up.

**Reply by vico93** [reply]
~~Thank you @SquidDev for the help. I was going to "risk" myself to try to do anything about it, but for other reasons i think i will revert permanently to 1.12 due to frustration about other things not related to CC:Tweaked.~~

But again, thank you to maintain this mod, i can't play Minecraft without it!

**Reply by vico93** [reply]
So, i tried a little endevour, cloned the repo and only changed the version number in `gradle.properties` and tried to build using the command you stated above.

Obviously it doesn't worked, but due to the fact that Forge still doesn't have a build for 1.20 pre releases. So i dont know how much of the code will error when trying to build straight to 1.20.

**Comment by vico93** [comment]
So i went ahead and tried to boot CC:T straight into 1.20-pre7 to see what error it would produce.

Got this [crash report](https://gist.github.com/vico93/8e0651d70b2143a6f6c009e6176f9eff).

Seems that an update would require far more knowledge that i do, unfortunately. Maybe what that report contains could already give a hint to @SquidDev of what will be need to be changed when he decides to update it.

**Reply by SirEdvin** [reply]
This probably means that you need to modify some code to 1.20-pre7. Probably some class was renamed, moved or replaced with another one.
